Modules/Loadable/Transforms/SubjectHierarchyPlugins/qSlicerSubjectHierarchyTransformsPlugin.cxx

```diff
@@ -408,7 +408,9 @@ void qSlicerSubjectHierarchyTransformsPlugin::invert()
     shNode->GetItemDataNode(currentItemID) );
   if (transformNode)
     {
+    MRMLNodeModifyBlocker blocker(transformNode);
     transformNode->Inverse();
+    transformNode->InverseName();
```

Comment:
Wouldn't it make more sense to put the `InverseName` inside the `Inverse` method of the node rather than having duplicate logic in the GUI code.  Also if anyone outside uses the `Inverse` method they will get consistent results.

Reply:
I've though about that, too. In fact, I've implemented like that initially. But it then I moved it out to a separate function to not change current behavior, as modules may not expect this side effect. For example, if Inverse() would include InverseName() then result of SetName(); Inverse(); would differ from Inverse(); SetName().

I talked to @ungi about this, too, and he preferred keeping these two methods separate (at MRML level).